cypress/e2e/hub/collections.cy.ts

```diff
@@ -18,9 +18,10 @@ describe.skip('Collections- List View', () => {
   });
 
   it('user can upload and delete collection', () => {
-    const namespace = `upload_namespace_${randomString(3, undefined, { isLowercase: true })}`;
+    const namespace = 'hub_e2e_namespace_' + randomString(3, undefined, { isLowercase: true });
     cy.createNamespace(namespace);
-    const collection = randomString(5, undefined, { isLowercase: true }).replace(/\d/g, '');
+    const collection =
+      'hub_e2e_collection_' + randomString(5, undefined, { isLowercase: true }).replace(/\d/g, '');
     cy.galaxykit(`collection upload ${namespace} ${collection} --skip-upload`).then((result) => {
```

Comment:
There should be same cy.galaxykit('task wait all').

Every time you are doing some galaxykit command that modifies data (namespace create, collection upload...), you need to call task wait all. Because galaxykit returns when task is queued in the server, not when the task is finish.

But galaxykit task wait all returns when the server has no running tasks, so you can be sure its done. 

Also. you dont have to call then after galaxykit. It could be easily as:
cy.galaxykit('collection upoload...');
cy.galaxkit('task wait all');
cy.navigateTo('hub', Collections.url);
....

it is the same and its more readable.